s2a/build.gradle

```diff
@@ -11,6 +11,7 @@ plugins {
 description = "gRPC: S2A"
 
 dependencies {
+    implementation 'com.google.s2a.proto.v2:s2a-proto:0.1.1'
```

Comment:
Add this to `gradle/libs.versions.toml` and reference it here using `libraries.` like below.